api-docs/.eleventy.js

```diff
@@ -0,0 +1,75 @@
+import syntaxHighlightPlugin from '@11ty/eleventy-plugin-syntaxhighlight';
+import markdownIt from 'markdown-it';
+import markdownItAnchor from 'markdown-it-anchor';
+import { parse } from 'node-html-parser';
+
+export default function (config) {
+  // See https://www.11ty.dev/docs/config/
+  config.setInputDirectory('content');
+  config.setOutputDirectory('../public');
+  config.setDataDirectory('../_data');
+  config.setIncludesDirectory('../_includes');
+
+  // See https://www.11ty.dev/docs/languages/markdown/
+  config.setLibrary(
+    'md',
+    markdownIt({
+      html: true,
+      linkify: true,
+    }).use(markdownItAnchor, {
+      slugify: config.getFilter('slugify'),
+      level: [1, 2, 3, 4],
+      permalink: markdownItAnchor.permalink.ariaHidden({
+        placement: 'after',
+        class: 'anchor-link',
+        symbol: '#',
+      }),
+    }),
+  );
+
+  config.addPlugin(syntaxHighlightPlugin, {
+    preAttributes: {
+      class: ({ language }) => `code-block language-${language}`,
+    },
+  });
+
+  config.addPassthroughCopy('css');
+  config.addPassthroughCopy('js');
+  config.addPassthroughCopy({ 'img/favicon.ico': '/' });
```

Comment:
This fails when you run `yarn api-docs:start` the first time when there is no `public` folder:
```
[11ty] Error with passthrough copy:
[11ty] 1. Having trouble copying './img/favicon.ico' (via TemplatePassthroughManagerCopyError)
[11ty] 2. Error copying passthrough files: ENOTEMPTY: directory not empty, rmdir '../public' (via TemplatePassthroughError)
[11ty] 3. ENOTEMPTY: directory not empty, rmdir '../public'
```

Reply:
Fixed.